markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+sponsors, we run events such as theatre trips, talks, and our infamous bar
+nights (psst, all drinks are on DoCSoc)! Our membership base is mostly comprised
+of Computing, JMC, and EIE students, but membership can also be purchased on the
+[union
+website](https://www.imperialcollegeunion.org/activities/a-to-z/computing) for
+anyone not on a DoC course. Why not get your friends on other courses to become
+members too?
+
+To stay up-to-date on the events that DoCSoc runs, keep an eye out for the
+weekly digest (you're reading it right now) which is baked and then delivered
+fresh to your inbox every week! Additionally, liking our [Facebook
+page](https://www.facebook.com/ICDocSoc/) will help you stay up to date on
+everything going on with DoCSoc.
+
+## Our Sponsors
+Without the support of our generous sponsors, much of what we do and what makes
+DoCSoc so special would not be possible! Sponsors host many of our talks and
+will occasionally let you know of their great job opportunities.
+
+Which brings me on to why DoCSoc gets sponsorship: It's important to realise
+that our sponsors are looking to hire YOU! You get to build your career while we
+get the money to run awesome social events. To kick things off, we will be
+hosting our annual Sponsors' Exhibition on the 12th October from 14.00-17.30 in
+the Great Hall (more details below).
+
+## Our Main Events
+
+### Bar Nights
+
+Oh, bar nights&hellip; Arguably one of DoCSoc's most known type of event, bar nights
+happen just about every month during term time. They're a great opportunity to
+talk to your coursemates and make new friends! Upon arrival, a volunteer will
+greet you and hand you tokens which can be redeemed for a vast array of drinks
+(both alcoholic and non-alcoholic) behind the bar. Ran out of tokens? Depending
+on the event, just ask a committee member to give you a few more tokens. We
+don't bite!
+
+### Board Game Nights
+
+Whether you're a *Monopoly* person or a *Settlers of Catan* person, board game
+nights are the perfect opportunity to get your nerd on and fight for victory
+with free snacks and drinks all evening.
+
+### Talks
+Our talks usually happen in the evenings, just after lectures. If you attended a
+talk around lunch time and didn't like it, you're most likely confusing the
+department's ACI talks with our talks. At the end of a talk we usually hand out
+pizza for everyone! Veggie options are of course available.
+
+### IC Hack
+We run a hackathon every year called IC Hack.
+
+Check out the epicness of IC Hack 2017 on
+[YouTube](https://www.youtube.com/watch?v=VQl8tLcxptY)!
+
+Want to get participate? Want to help run IC Hack? Details will follow later this
+term!
+We encourage all freshers to participate in their first year. Hackathons are a
+blast to attend and an amazing experience regardless of whether you've never
+been to a hackathon before or attend a different one very week.
+
+---
+
+Now for all the juicy details on our events this week(ish):
```

Comment:
not just our events (CodeCon)

Reply:
They are still "our" events, in that we host the sponsor. Tomato/tomato.